main/src/addins/MacPlatform/MacPlatform.cs

```diff
@@ -370,10 +370,22 @@ static void NewInstanceMenuItem_Activated (object sender, EventArgs e)
 				LoggingService.LogError ($"Failed to start new instance: {error.LocalizedDescription}");
 		}
 
-
+		// The Enabled key needs to be controlled through NSUserDefaults so that it can be
+		// set from the command line. This lets users who require accessibility features to
+		// enabled to control it through the commandline, if it has been disabled from inside the software
+		static bool voiceOverNoticeShown;
 		const string EnabledKey = "com.monodevelop.AccessibilityEnabled";
+		const string VoiceOverNoticeShownKey = "com.monodevelop.VoiceOverShown";
```

Comment:
Nit: Key naming style in propertyservice is like:
```suggestion
const string VoiceOverNoticeShownKey = "MonoDevelop.VoiceOver.Shown";
```